src/main/java/seedu/address/logic/commands/SetProjectStatusCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a task to a project.
+ */
+public class SetProjectStatusCommand extends SetStatusCommand {
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " [complete/incomplete] /to PROJECT_NAME";
+
+    public static final String MESSAGE_SUCCESS = "The project %1$s is set as %2$s.";
+
+    public static final String MESSAGE_PROJECT_NOT_FOUND = "Project %1$s not found: "
+            + "Please make sure the project exists.";
+
+    private final Person project;
```

Comment:
these would have to be changed according to the new refactored code project, the message strings look good and are consistent with the general layout